go-controller/pkg/cni/bandwidth.go

```diff
@@ -87,11 +89,17 @@ func getPodBandwidth(ifname string) (ingressBPS, egressBPS int64, err error) {
 	}
 	// egressBPS
 	out, err = ovsGet("interface", ifname, "ingress_policing_rate", "")
-	if err == nil && len(out) > 0 {
-		egressBPS, err = strconv.ParseInt(out, 10, 64)
-		if err == nil {
-			egressBPS *= 1000
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "failed to get ingress_policing_rate for interface %s", ifname)
+	}
+	if len(out) > 0 {
+		egressValue, err := strconv.ParseInt(out, 10, 64)
+		if err != nil {
+			return 0, 0, errors.Wrapf(err, "failed to parse ingress_policing_rate %s", out)
+		}
+		if egressValue != 0 { // 0 is the default value so we return as it was not found
+			egressBPS = egressValue * 1000
```

Comment:
@roytman why do we determine egressValue from ingress?

Can we add a comment here to say the reason, otherwise this isn't obvious

Reply:
That's commented here where we use it https://github.com/ovn-org/ovn-kubernetes/blob/9ce0b1fbc70b8cbaa2232b66c10adeb00906babf/go-controller/pkg/cni/bandwidth.go#L40

I'll drop the same comment